os-eap-launch/added/launch/resource-adapters-common.sh

```diff
@@ -106,7 +107,13 @@ function inject_resource_adapters_common() {
       resource_adapter="${resource_adapter}<transaction-support>$transaction_support</transaction-support>"
     fi
 
-    resource_adapter="${resource_adapter}<connection-definitions><connection-definition class-name=\"${ra_class}\" jndi-name=\"${ra_jndi}\" enabled=\"true\" use-java-context=\"true\">"
+    resource_adapter="${resource_adapter}<connection-definitions><connection-definition"
+    # monitor applications, look for unclosed resources.
+    tracking=$(find_env "${ra_prefix}_TRACKING" "true")
```

Comment:
1. I think this variable can be declared as local, and moved into the "if [[ "$JBOSS_EAP_VERSION" == "7."* ]]" statement below it.
2. Shouldn't the default be "false", per the workaround described in CLOUD-2455?

Reply:
agreed, thanks.